src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,85 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, generateReport(readContent(fromFileName)));
+    }
+
+    private String readContent(String fromFileName) {
+        final StringBuilder stringBuilder = new StringBuilder();
+        String value;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            value = reader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Cant read file" + fromFileName, e);
+        }
+        return stringBuilder.toString();
+    }
+
+    private ArrayList<String> generateReport(String rawContent) {
+        String[] receivedData = rawContent.split(System.lineSeparator());
+        ArrayList<String> report = new ArrayList<>();
```

Comment:
```suggestion
        List<String> report = new ArrayList<>();
```
хорошою практикою вважається використання посилання інтерфейсу, щоб код був більш гнучким